python311Packages.prettytable: 3.5.0 -> 3.8.0

fabaff opened the pull request:
Diff: https://github.com/jazzband/prettytable/compare/refs/tags/3.5.0...3.8.0

Changelog: https://github.com/jazzband/prettytable/releases/tag/3.8.0

## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

NickCao commented on pkgs/development/python-modules/prettytable/default.nix:
```diff
@@ -1,27 +1,26 @@
   format = "pyproject";
 
-  disabled = pythonOlder "3.7";
+  disabled = pythonOlder "3.";
```
```suggestion
  disabled = pythonOlder "3.8";
```

fabaff commented:
Result of `nixpkgs-review pr 251813` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>28 packages marked as broken and skipped:</summary>
  <ul>
    <li>libvmi</li>
    <li>python310Packages.optuna</li>
    <li>python310Packages.optuna.dist</li>
    <li>python310Packages.oslo-db</li>
    <li>python310Packages.oslo-db.dist</li>
    <li>python310Packages.subunit2sql</li>
    <li>python310Packages.subunit2sql.dist</li>
    <li>python311Packages.optuna</li>
    <li>python311Packages.optuna.dist</li>
    <li>python311Packages.oslo-db</li>
    <li>python311Packages.oslo-db.dist</li>
    <li>python311Packages.subunit2sql</li>
    <li>python311Packages.subunit2sql.dist</li>
    <li>qemu_xen</li>
    <li>qemu_xen.debug</li>
    <li>qemu_xen.ga</li>
    <li>qemu_xen_4_15</li>
    <li>qemu_xen_4_15.debug</li>
    <li>qemu_xen_4_15.ga</li>
    <li>qubes-core-vchan-xen</li>
    <li>xen</li>
    <li>xen-slim</li>
    <li>xenPackages.xen-slim</li>
    <li>xenPackages.xen-vanilla</li>
    <li>xenPackages.xen_4_15-slim</li>
    <li>xenPackages.xen_4_15-vanilla</li>
    <li>xen_4_15</li>
    <li>xen_4_15-slim</li>
  </ul>
</details>
<details>
  <summary>6 packages failed to build:</summary>
  <ul>
    <li>dduper</li>
    <li>echidna</li>
    <li>python310Packages.insightface</li>
    <li>python310Packages.insightface.dist</li>
    <li>python311Packages.insightface</li>
    <li>python311Packages.insightface.dist</li>
  </ul>
</details>
<details>
  <summary>204 packages built:</summary>
  <ul>
    <li>ceph-client</li>
    <li>ceph-csi</li>
    <li>ceph-dev</li>
    <li>ceph-dev.dev</li>
    <li>ceph-dev.doc</li>
    <li>libceph (ceph-dev.lib ,libceph.dev ,libceph.doc ,libceph.lib ,libceph.man)</li>
    <li>ceph-dev.man</li>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>creds</li>
    <li>creds.dist</li>
    <li>glanceclient (python310Packages.python-glanceclient)</li>
    <li>glanceclient.dist (python310Packages.python-glanceclient.dist)</li>
    <li>glusterfs</li>
    <li>heatclient (python310Packages.python-heatclient)</li>
    <li>heatclient.dist (python310Packages.python-heatclient.dist)</li>
    <li>home-assistant-component-tests.honeywell</li>
    <li>home-assistant-component-tests.nx584</li>
    <li>hstsparser</li>
    <li>hstsparser.dist</li>
    <li>imdshift</li>
    <li>imdshift.dist</li>
    <li>ironicclient (python310Packages.python-ironicclient)</li>
    <li>ironicclient.dist (python310Packages.python-ironicclient.dist)</li>
    <li>jenkins-job-builder (python310Packages.jenkins-job-builder)</li>
    <li>jenkins-job-builder.dist (python310Packages.jenkins-job-builder.dist)</li>
    <li>kube-hunter</li>
    <li>kube-hunter.dist</li>
    <li>lexicon</li>
    <li>lexicon.dist</li>
    <li>manilaclient (python310Packages.python-manilaclient)</li>
    <li>manilaclient.dist (python310Packages.python-manilaclient.dist)</li>
    <li>openstackclient (python310Packages.python-openstackclient)</li>
    <li>openstackclient.dist (python310Packages.python-openstackclient.dist)</li>
    <li>patroni</li>
    <li>patroni.dist</li>
    <li>polygon-cli</li>
    <li>polygon-cli.dist</li>
    <li>pynitrokey</li>
    <li>pynitrokey.dist</li>
    <li>pyocd (python310Packages.pyocd)</li>
    <li>pyocd.dist (python310Packages.pyocd.dist)</li>
    <li>python310Packages.aigpy</li>
    <li>python310Packages.aigpy.dist</li>
    <li>python310Packages.aiosomecomfort</li>
    <li>python310Packages.aiosomecomfort.dist</li>
    <li>python310Packages.cliff</li>
    <li>python310Packages.cliff.dist</li>
    <li>python310Packages.duckdb-engine</li>
    <li>python310Packages.duckdb-engine.dist</li>
    <li>python310Packages.fiblary3-fork</li>
    <li>python310Packages.fiblary3-fork.dist</li>
    <li>python310Packages.hacking</li>
    <li>python310Packages.hacking.dist</li>
    <li>python310Packages.ibis-framework</li>
    <li>python310Packages.ibis-framework.dist</li>
    <li>python310Packages.ipython-sql</li>
    <li>python310Packages.ipython-sql.dist</li>
    <li>python310Packages.keystoneauth1</li>
    <li>python310Packages.keystoneauth1.dist</li>
    <li>python310Packages.ldappool</li>
    <li>python310Packages.ldappool.dist</li>
    <li>python310Packages.openstacksdk</li>
    <li>python310Packages.openstacksdk.dist</li>
    <li>python310Packages.osc-lib</li>
    <li>python310Packages.osc-lib.dist</li>
    <li>python310Packages.oslo-concurrency</li>
    <li>python310Packages.oslo-concurrency.dist</li>
    <li>python310Packages.oslo-config</li>
    <li>python310Packages.oslo-config.dist</li>
    <li>python310Packages.oslo-context</li>
    <li>python310Packages.oslo-context.dist</li>
    <li>python310Packages.oslo-i18n</li>
    <li>python310Packages.oslo-i18n.dist</li>
    <li>python310Packages.oslo-log</li>
    <li>python310Packages.oslo-log.dist</li>
    <li>python310Packages.oslo-serialization</li>
    <li>python310Packages.oslo-serialization.dist</li>
    <li>python310Packages.oslo-utils</li>
    <li>python310Packages.oslo-utils.dist</li>
    <li>python310Packages.prettytable</li>
    <li>python310Packages.prettytable.dist</li>
    <li>python310Packages.pynx584</li>
    <li>python310Packages.pynx584.dist</li>
    <li>python310Packages.python-cinderclient</li>
    <li>python310Packages.python-cinderclient.dist</li>
    <li>python310Packages.python-jenkins</li>
    <li>python310Packages.python-jenkins.dist</li>
    <li>python310Packages.python-keystoneclient</li>
    <li>python310Packages.python-keystoneclient.dist</li>
    <li>python310Packages.python-novaclient</li>
    <li>python310Packages.python-novaclient.dist</li>
    <li>swiftclient (python310Packages.python-swiftclient)</li>
    <li>swiftclient.dist (python310Packages.python-swiftclient.dist)</li>
    <li>slither-analyzer (python310Packages.slither-analyzer)</li>
    <li>slither-analyzer.dist (python310Packages.slither-analyzer.dist)</li>
    <li>python310Packages.softlayer</li>
    <li>python310Packages.softlayer.dist</li>
    <li>python310Packages.spsdk</li>
    <li>python310Packages.spsdk.dist</li>
    <li>python310Packages.stestr</li>
    <li>python310Packages.stestr.dist</li>
    <li>python310Packages.tempest</li>
    <li>python310Packages.tempest.dist</li>
    <li>python310Packages.tern</li>
    <li>python310Packages.tern.dist</li>
    <li>python310Packages.weboob</li>
    <li>python310Packages.weboob.dist</li>
    <li>python310Packages.woob</li>
    <li>python310Packages.woob.dist</li>
    <li>python311Packages.aigpy</li>
    <li>python311Packages.aigpy.dist</li>
    <li>python311Packages.aiosomecomfort</li>
    <li>python311Packages.aiosomecomfort.dist</li>
    <li>python311Packages.cliff</li>
    <li>python311Packages.cliff.dist</li>
    <li>python311Packages.duckdb-engine</li>
    <li>python311Packages.duckdb-engine.dist</li>
    <li>python311Packages.fiblary3-fork</li>
    <li>python311Packages.fiblary3-fork.dist</li>
    <li>python311Packages.hacking</li>
    <li>python311Packages.hacking.dist</li>
    <li>python311Packages.ibis-framework</li>
    <li>python311Packages.ibis-framework.dist</li>
    <li>python311Packages.ipython-sql</li>
    <li>python311Packages.ipython-sql.dist</li>
    <li>python311Packages.jenkins-job-builder</li>
    <li>python311Packages.jenkins-job-builder.dist</li>
    <li>python311Packages.keystoneauth1</li>
    <li>python311Packages.keystoneauth1.dist</li>
    <li>python311Packages.ldappool</li>
    <li>python311Packages.ldappool.dist</li>
    <li>python311Packages.openstacksdk</li>
    <li>python311Packages.openstacksdk.dist</li>
    <li>python311Packages.osc-lib</li>
    <li>python311Packages.osc-lib.dist</li>
    <li>python311Packages.oslo-concurrency</li>
    <li>python311Packages.oslo-concurrency.dist</li>
    <li>python311Packages.oslo-config</li>
    <li>python311Packages.oslo-config.dist</li>
    <li>python311Packages.oslo-context</li>
    <li>python311Packages.oslo-context.dist</li>
    <li>python311Packages.oslo-i18n</li>
    <li>python311Packages.oslo-i18n.dist</li>
    <li>python311Packages.oslo-log</li>
    <li>python311Packages.oslo-log.dist</li>
    <li>python311Packages.oslo-serialization</li>
    <li>python311Packages.oslo-serialization.dist</li>
    <li>python311Packages.oslo-utils</li>
    <li>python311Packages.oslo-utils.dist</li>
    <li>python311Packages.prettytable</li>
    <li>python311Packages.prettytable.dist</li>
    <li>python311Packages.pynx584</li>
    <li>python311Packages.pynx584.dist</li>
    <li>python311Packages.pyocd</li>
    <li>python311Packages.pyocd.dist</li>
    <li>python311Packages.python-cinderclient</li>
    <li>python311Packages.python-cinderclient.dist</li>
    <li>python311Packages.python-glanceclient</li>
    <li>python311Packages.python-glanceclient.dist</li>
    <li>python311Packages.python-heatclient</li>
    <li>python311Packages.python-heatclient.dist</li>
    <li>python311Packages.python-ironicclient</li>
    <li>python311Packages.python-ironicclient.dist</li>
    <li>python311Packages.python-jenkins</li>
    <li>python311Packages.python-jenkins.dist</li>
    <li>python311Packages.python-keystoneclient</li>
    <li>python311Packages.python-keystoneclient.dist</li>
    <li>python311Packages.python-manilaclient</li>
    <li>python311Packages.python-manilaclient.dist</li>
    <li>python311Packages.python-novaclient</li>
    <li>python311Packages.python-novaclient.dist</li>
    <li>python311Packages.python-openstackclient</li>
    <li>python311Packages.python-openstackclient.dist</li>
    <li>python311Packages.python-swiftclient</li>
    <li>python311Packages.python-swiftclient.dist</li>
    <li>python311Packages.slither-analyzer</li>
    <li>python311Packages.slither-analyzer.dist</li>
    <li>python311Packages.softlayer</li>
    <li>python311Packages.softlayer.dist</li>
    <li>python311Packages.spsdk</li>
    <li>python311Packages.spsdk.dist</li>
    <li>python311Packages.stestr</li>
    <li>python311Packages.stestr.dist</li>
    <li>python311Packages.tempest</li>
    <li>python311Packages.tempest.dist</li>
    <li>python311Packages.tern</li>
    <li>python311Packages.tern.dist</li>
    <li>python311Packages.weboob</li>
    <li>python311Packages.weboob.dist</li>
    <li>python311Packages.woob</li>
    <li>python311Packages.woob.dist</li>
    <li>qemu_full</li>
    <li>qemu_full.debug</li>
    <li>qemu_full.ga</li>
    <li>quark-engine</li>
    <li>quark-engine.dist</li>
    <li>sambaFull (samba4Full)</li>
    <li>sambaFull.dev (samba4Full.dev)</li>
    <li>sambaFull.man (samba4Full.man)</li>
    <li>sigma-cli</li>
    <li>sigma-cli.dist</li>
    <li>tidal-dl</li>
    <li>tidal-dl.dist</li>
  </ul>
</details>

fabaff committed:
python311Packages.prettytable: 3.5.0 -> 3.8.0

Diff: jazzband/prettytable@refs/tags/3.5.0...3.8.0

Changelog: https://github.com/jazzband/prettytable/releases/tag/3.8.0
